react/style-guide.md

```diff
@@ -41,30 +41,36 @@ src
 └───app
 │   │
 │   └───components
-│   | └───BaseComponents
-│   | | └───Input
-│   | | └───Text
-│   | | └───Button
-│   | | └───etc
+│   │ └───baseComponents
+│   │     └───Input
+│   │     └───Text
+│   │     └───Button
+│   │     └───etc
 │   └───screens
-| | └───aScreenComponent
-| | | └───components
-| | | | constants.js
-| | | | i18n.js
-| | | | index.js
-| | | | layout.js
-| | | | styles.scss
-| | | | utils.js
-└───assets
+│       └───MyScreenComponent
+│           └───assets // Screen specific app assets
+│           | components
+│           | constants.js
+│           | i18n.js
+│           | index.js
+│           | layout.js
+│           | styles.scss
+│           | utils.js
+│
+└───assets // General app assets
```

Comment:
I'd move into `assets` into `app` following that each `assets` folder belongs to its screen and its screen childs, `app` would be the father of all the screen and the most general level.

Reply:
I'd also mention that every asset should be inside a folder.
e.g. `close` could contain `close.png`, `close@2x.png` and `close@3x.png`. Same for `active/inactive` assets.

Reply:
That only applies to RN. I'd aadd that to specific RN styleguides

Reply:
Also, I'm not entirely sure I agree on having an assets folder per component. I'm not sure what it adds, and a lot of assets are shared thougout the app.

Reply:
Not per component. Per screen